src/seedu/addressbook/AddressBook.java

```diff
@@ -486,9 +479,19 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
+            for(Iterator iterator = keywords.iterator(); iterator.hasNext();){
```

Comment:
According to the style guide, there should always be a space between the `for` and the opening parenthesis.